src/App.tsx

```diff
@@ -47,132 +49,132 @@ const HorizontalList = styled.div`
     padding: 20px;
   }
 `
-const AppSelector = (props: {applications: any}) => {
-
-    const {applications} = props
-    const history = useHistory()
-    const links = Object.values(applications).map((setting) => (
-        <div key={setting.name}>
-            <CardWrapper onClick={() => history.push(`/${setting.urlPath}`)}>
-                <CardHeader>
-                    <CardHeading>{`${setting.label}`}</CardHeading>
-                </CardHeader>
-                <CardBody>
-                    <CardFieldset>{`${setting.description}`}</CardFieldset>
-                </CardBody>
-            </CardWrapper>
-        </div>
-    ))
-    return (
-        <div>
-            <HorizontalList>{links}</HorizontalList>
-        </div>
-    )
+const AppSelector = (props: { applications: any }) => {
+  const { applications } = props
+  const history = useHistory()
+  const links = Object.values(applications).map((setting) => (
+    <div key={setting.name}>
+      <CardWrapper onClick={() => history.push(`/${setting.urlPath}`)}>
+        <CardHeader>
+          <CardHeading>{`${setting.label}`}</CardHeading>
+        </CardHeader>
+        <CardBody>
+          <CardFieldset>{`${setting.description}`}</CardFieldset>
```

Comment:
Think this should follow the "new bootstrapper standard" now. Meaning no "portal stuff